src/store/inventory-actions.js

```diff
@@ -170,11 +170,13 @@ export const toggleTagModal = (isOpen) => ({
     payload: { isOpen }
 });
 
-export const fetchAllTags = (search, pagination, getTags = defaultGetAllTags) => ({
-    type: ACTION_TYPES.ALL_TAGS,
-    payload: getTags(search, pagination),
-    meta: { lastDateRequestTags: Date.now() }
-});
+export const fetchAllTags = (search, pagination, getTags = defaultGetAllTags) => {
+    return {
```

Comment:
We should revert this to the implicit return as it was before to keep the same form as the other actions.

Reply:
Yep, that's a clean up I forgot to do, was checking the values of these parameters and forgot to change it back